static/app/views/organizationGroupDetails/header.tsx

```diff
@@ -428,7 +425,7 @@ class GroupHeader extends Component<Props, State> {
             query={location.query}
           />
           <NavTabs>
-            {isPerformanceIssue
+            {group.issueCategory === 'performance'
```

Comment:
I think we should use `IssueCategory.PERFORMANCE` instead here

Reply:
oops yeah